controllers/cardsController.js

```diff
@@ -119,8 +119,21 @@ exports.getCardDetailPage = async function(req, res, next) {
 		}
 
 		var stats = await getCardStats(req.user, card);
-		res.render('cardDetail', {
-			title: i18next.t("lang")==="en"?cardData.name:cardData.ja_name,
+
+		var title, lang = i18next.t("lang");
+		if (lang === "en") {
+			title = cardData.name;
+		} else if (lang === "ja") {
+			title = cardData.ja_name;
+		} else if (lang === "zh") {
+			title = cardData.zh_name;
+		}
```

Comment:
If-else can be removed if the translations are put in a map
`card: {name: {en: "name", ja: "???", zh: "???"}}`
`title = cardData.name[lang]`